.github/workflows/release-minor.yml

```diff
@@ -18,13 +17,31 @@ jobs:
       - name: Push to master
         uses: ad-m/github-push-action@master
         with:
-          github_token: ${{ secrets.GH_RW_TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
           branch: refs/heads/master
 
-      - name: Success Message
-        run: echo "Release successful. Check out https://github.com/Workday/canvas-kit/actions/workflows/release.yml to see the release job start."
+      - name: Checkout master
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+          fetch-depth: 0 # Needed to do a proper push
+          ref: master # Release job only works on master
 
-      - name: Failure Message
-        if: failure()
-        run: echo "Make sure there are no changes https://github.com/Workday/canvas-kit/compare/prerelease/minor...master and try again."
+      - uses: Workday/canvas-kit-actions/install@v1
+        with:
+          node_version: 14.x
```

Comment:
`Workday/canvas-kit-actions/install@v1` is composite action doing all caching and node install actions.

Reply:
yes, it cleans our actions a lot. for example, we have 9 places in workflows where we use catching or node install. It's about 30 lines of code. Now we can change all of them by calling composite action `install`.So actions would be cleaner and it would be easier to maintain.

Reply:
[In this issue](https://github.com/Workday/canvas-kit/issues/2107) you can get more info about this action and why do we need it.